client/components/course/Outline/InsertActivity/SelectAction.vue

```diff
@@ -16,11 +16,17 @@ export default {
   computed: {
     actions() {
       const select = action => () => this.$emit('selected', action);
-      return [
+      const actions = [
         { label: 'Create', icon: 'plus', action: select('create') },
         { label: 'Copy', icon: 'content-copy', action: select('copy') },
         { label: 'Cancel', icon: 'close', action: () => this.$emit('close') }
       ];
+
+      if (!process.env.ENABLE_ACTIVITY_LINKING) return actions;
+      actions.splice(
+        2, 0, { label: 'Use existing', icon: 'link', action: select('link') },
+      );
+      return actions;
```

Comment:
Hmm, you can make it more compact relying on object truthiness:
```js
const isLinkingSupported = process.env.ENABLE_ACTIVITY_LINKING;

const actions = [
  { label: 'Create', icon: 'plus', action: select('create') },
  { label: 'Copy', icon: 'content-copy', action: select('copy') },
  isLinkingSupported && 
    { label: 'Use existing', icon: 'link', action: select('link') },
  { label: 'Cancel', icon: 'close', action: () => this.$emit('close') }
].filter(Boolean);
```

Reply:
Me can do that. 🐒